apps/meteor/ee/server/apps/cron.js

```diff
@@ -100,13 +96,13 @@ const appsUpdateMarketplaceInfo = Meteor.bindEnvironment(function _appsUpdateMar
 		Apps.debugLog(err);
 	}
 
-	Promise.await(Apps.updateAppsMarketplaceInfo(data).then(notifyAdminsAboutInvalidApps).then(notifyAdminsAboutRenewedApps));
+	await Apps.updateAppsMarketplaceInfo(data).then(notifyAdminsAboutInvalidApps).then(notifyAdminsAboutRenewedApps);
 });
 
 SyncedCron.add({
 	name: 'Apps-Engine:check',
 	schedule: (parser) => parser.text('at 4:00 am'),
-	job() {
-		appsUpdateMarketplaceInfo();
+	async job() {
+		await appsUpdateMarketplaceInfo();
```

Comment:
are you sure this is going to work?

Reply:
It should, we have other async functions on synced crons, some from 2 years ago 👀